src/Tutorial/ui/DocumentationRoot.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import Button from "@mui/material/Button";
+import { MD } from "../../ui/MD/MD";
+
+import { getPage } from "./root";
+import { Navigator, useHistory } from "../../ui/React/Documentation";
+
+const resolveRelativePath = (folder: string, relative: string): string => {
```

Comment:
Pretty sure we already have utility functions for these.

Reply:
We do but there is a little finessing needed to get the existing resolveFilePath to work here, since it defaults to using an absolute path unless the path actually starts with `.`

I'll post a suggestion that would use the existing filepath functions